package/gluon-core/luasrc/lib/gluon/upgrade/010-primary-mac

```diff
@@ -28,6 +28,8 @@ if platform.match('ar71xx', 'generic', {'tl-wdr3600', 'tl-wdr4300',
   table.insert(try_files, 1, '/sys/class/ieee80211/phy1/macaddress')
 elseif platform.match('ramips', 'mt7621', {'dir-860l-b1'}) then
   table.insert(try_files, 1, '/sys/class/ieee80211/phy1/macaddress')
+elseif platform.match('ramips', 'mt7620', {'miwifi-mini'}) then
+  table.insert(try_files, 1, '/sys/class/net/eth0/address')
```

Comment:
are you sure this is the only way, not like with mt7621 ?

Reply:
And mt7620 before mt7621.

Reply:
i tried the other ways, phy1/macaddress is off by one in the last digit, only the address on eth0 is matching the one on the case.